c/include/prima/prima.h

```diff
@@ -202,26 +202,28 @@ PRIMAC_API
 int prima_free_result(prima_result * result);
 
 /*
+ * Algorithm
+ */
+typedef enum
+{
+  PRIMA_BOBYQA,
+  PRIMA_COBYLA,
+  PRIMA_LINCOA,
+  PRIMA_NEWUOA,
+  PRIMA_UOBYQA
+} prima_algorithm;
```

Comment:
Actually one more comment, can we append `_t` to this? I've often used `_t` for all typedefs to make to clear when reading the code that I'm looking at a type and not a value. I'll make another PR to add `_t` to the various other structs defined here (`prima_options` and `prima_objcon`, etc.).

Reply:
good idea